prow/prowjobs/GoogleCloudPlatform/esp-v2/esp-v2.yaml

```diff
@@ -52,7 +52,9 @@ presubmits:
     always_run: true
     decorate: true
     labels:
+      preset-k8s-ssh: "true"
       preset-service-account: "true"
+      preset-dind-enabled: "true"
```

Comment:
this doesn't match the title. are you actually using docker in docker? what for?

Reply:
Just keep all the config with our gob prow config

Reply:
yep, I think so.